packages/utils/src/getLogsRetryHelper.ts

```diff
@@ -24,6 +24,8 @@ export type GetLogsRetryHelperReturnType =
       shouldRetry: true;
       /** Suggested values to use for (fromBlock, toBlock) in follow-up eth_getLogs requests. */
       ranges: { fromBlock: Hex; toBlock: Hex }[];
+      /** `true` if the error message suggested to use this range on retry. */
+      isSuggestedRange: boolean;
```

Comment:
I wonder if instead of a boolean, this should be an optional number equal to a "confirmed" static range. Eg for a Cloudflare error, it would return `staticMaxRange: 800`.